### PR TITLE
added an example hello world

### PR DESCRIPTION
**Pull request by mika666**

### CODE REVIEW

File: HelloWorld/Program.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        {
+            Console.WriteLine("HelloWorld :: V1");
+            Console.WriteLine("In order to exit this program, type exit or stop");
+            Console.WriteLine();
+
+            while (true)
+            {
+                Console.WriteLine("Hi, what is your name?");
+
+                var name = Console.ReadLine().Trim();
```
Comment (bruniasty333) [inline HelloWorld/Program.cs:17]: o kurcze, eeeeee

File: HelloWorld/Program.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                }
+                else
+                {
+                    Console.WriteLine($"You are not Bruno :( But it is nice to see you anyway, {name}");
+                }
+
+                Console.WriteLine();
+            }
+
+            Console.WriteLine("Good bye!");
```
Comment (bruniasty333) [inline HelloWorld/Program.cs]: i would prefer to say night, night

**Review by bruniasty333 — CHANGES_REQUESTED** [review]

File: HelloWorld/Program.cs (hunk 1 above)
Comment (mika666) [reply HelloWorld/Program.cs:17]: no nie ma mowy!

File: HelloWorld/Program.cs (hunk 2 above)
Comment (mika666) [reply HelloWorld/Program.cs]: ok, a change submitted

**Review by bruniasty333 — APPROVED** [review]

**Pull request merged**